kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flight/AvailableFlight.java

```diff
@@ -1,2 +1,20 @@
-package com.kodilla.good.patterns.flight;public class AvailableFlight {
+package com.kodilla.good.patterns.flight;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AvailableFlight {
+
+    public static Map<Integer, Flight> getAvailableFlightMap() {
+        Map<Integer, Flight> availableFlightMap = new HashMap<>();
+        availableFlightMap.put(0,new Flight("Oslo Torp", "", "Poznan"));
+        availableFlightMap.put(1,new Flight("Wroclaw", "Warsaw", "Rzeszow"));
```

Comment:
Nie o to chodzi w lotach z przesiadkami, żeby z góry je ustawić (nie ma wtedy żadnego wyzwania w znajdywaniu lotu przesiadkowego ;).
Pomyśl o tym jak o prawdziwej wyszukiwarce - w bazie danych masz pojedyncze loty (np. Oslo Torp - Poznań i Warszawa - Rzeszów), a to wyszukiwarka musi znaleźć kombinację, składając 2 lub więcej z nich w lot przesiadkowy.
Czyli: na poziomie bazy lotów powinny być tylko loty bezpośrednie, a w wynikach wyszukiwarki (wartości zwracane z metod `find...` - obiekty umożliwiające reprezentację zarówno lotu bezpośredniego, jak i łączonego).